### ISSUE TITLE
The Menubar is not visible in mobile devices

### ISSUE THREAD

**Issue by Abhishek-465** [apu52/METAVERSE#645]
**Describe the bug**

The Menubar is not visible in mobile devices.

**Expected behavior**
It needs some adjustments in CSS.Apart from this the tagline in the hero section is also not readable in mobile devices. I want to make these changes and make the UI perfect in mobile devices.

**Screenshots (Mandatory)**

![Screenshot (55)](https://github.com/apu52/METAVERSE/assets/127030695/fd0fb8d1-3a48-46b4-ac58-d28abf5a30ad)

**Additional context (Mandatory )**
Kindly assign me this issue.

***Are you contributing under any open-source program ?***

Yes under GSSOC.

**Comment by apu52** [comment]
@Abhishek-465 assigned

**Comment by Abhishek-465** [comment]
@apu i have made the PR but it was closed by someone i don't know at all. Is he a project maintainer?

**Comment by apu52** [comment]
> @apu i have made the PR but it was closed by someone i don't know at all. Is he a project maintainer?

Yes. he is project Maintainer .but Don't worry . I have reopened the PR .I will review this . Don't worry

**Comment by Abhishek-465** [comment]
Okay @apu52 
thank you so much.
